elements/pfe-modal/pfe-modal.ts

```diff
@@ -74,7 +73,7 @@ export class ModalOpenEvent extends ComposedEvent {
  * @cssprop {<color>} --pf-c-modal-box--BackgroundColor - {@default #fff}
  * @cssprop --pf-c-modal-box__title--FontFamily - default font family for header-slotted headings
  */
-@customElement('pfe-modal') @pfelement()
+@customElement('pfe-modal')
```

Comment:
Do we need `@pfelement()`?

Reply:
`@pfelement` does 3 things:
1. adds `PFElement` className and `pfelement` attr to the host element on connect
2. adds optional perf tracking
3. enqueues body autoreveal

(1) PF doesn't do, not sure why we'd need to here, and if it is needed, let the user add them in their HTML
(2) likewise, this isn't a pf4 thing, so why would we need it? we can solve that problem otherwise in rhds, I'd think, and leave `<pfe-*>` for UI
(3) same same, this is kind of an app concern, we don't need component code to get involved

Reply:
Gotcha. 👍